cfme/tests/services/test_config_provider_servicecatalogs.py

```diff
@@ -39,6 +41,7 @@ def catalog_item(appliance, request, provider, ansible_tower_dialog, catalog, jo
     return catalog_item
 
 
+@pytest.mark.uncollectif(versioncheck, reason='API V2 not supported since Tower 3.6.')
```

Comment:
@mshriver I guess you won't like me to use the `uncollectif`. Also, the import of `versioncheck` import is questionable. But it is working and was the easiest way to not test against API v1 with 3.6 tower.

Reply:
No its fine - but as commented above, is this reason correct? Isn't it that V1 isn't supported in Ansible 3.6+?